clairvoyance/client.py

```diff
@@ -55,6 +55,7 @@ async def post(
                     self._url,
                     json=gql_document,
                     proxy=self.proxy,
+                    verify_ssl=False
```

Comment:
There is an option to handle certification skipping, available here in code in the `self.disable_ssl_verify` property and with the aforementioned flags in CLI. The option is respected above (in line 47: `aiohttp.TCPConnector(verify_ssl=(not self.disable_ssl_verify))`)

Currently I am unable to make tests for the invalid SSL/TLS certifications, but the way it's done is deprecated, according to the [aiohttp documentation](https://docs.aiohttp.org/en/stable/client_reference.html):
> verify_ssl ([bool](https://docs.python.org/3/library/functions.html#bool)) –
perform SSL certificate validation for HTTPS requests (enabled by default). May be disabled to skip validation for sites with invalid certificates.
**Deprecated since version 2.3: Pass verify_ssl to ClientSession.get() etc.**

If you are still interested in this PR, I'd suggest to make a change that would respect the option, otherwise I plan to make another PR during the next weekend.